examples/cifar10_resnet.py

```diff
@@ -116,7 +149,8 @@
                        kernel_initializer='he_normal',
                        kernel_regularizer=l2(1e-4))(x)
         x = keras.layers.add([x, y])
-        x = Activation('relu')(x)
+        if version != 2:
+            x = Activation('relu')(x)
```

Comment:
Is the placement of the relu the only difference between v1 and v2?

Reply:
I reviewed the 2 papers again. Resnet v1 is accurate. I may have overlooked some details in Resnet v2. Thus, I revised Resnet v2 following the Torch implementation by K He. Basically, Resnet v1 is characterized by post activation ( 2 3x3 Conv-BN-ReLU) while Resnet v2 is pre-activation (1x1 BN-ReLU-Conv, 3x3 BN-ReLU-Conv, 1x1 BN-ReLU-Conv or known as bottleneck layer).  Resnet v1 number of parameters are approx the same in paper of K He.